src/app/context/FormContext/FormContextReducer.js

```diff
@@ -1,7 +1,13 @@
-import { dispatchEventsType } from './DispatchEventsTypes'
+import DISPATCH_EVENTS from './DispatchEventsTypes'
 
 const formContextReducer = (state, action) => {
   switch (action.type) {
+    case DISPATCH_EVENTS.SET_CURRENT_QUESTION_TO_STATE:
+      return { ...action.payload }
```

Comment:
Here will be an error if the payload is null... Please, provide an extra check or make some amendments